gum/gumdarwingrafter.c

```diff
@@ -666,6 +671,222 @@ gum_darwin_grafter_compute_layout (GumDarwinGrafter * self,
   }
 }
 
+static void
+gum_collect_chained_imports (GumDarwinModule * self,
```

Comment:
Since this isn't a method of the `DarwinModule` class, we shouldn't name it `self`.

```suggestion
gum_collect_chained_imports (GumDarwinModule * module,
```

(Or `mod` in case it requires breaking lines like this.)